INIT.ASM

```diff
@@ -36,7 +37,14 @@ endif
     mov [@@temp], ax
     fldcw [@@temp]
 
+     call __keyb_install_keyboard_handler
```

Comment:
Please rename `__keyb_install_keyboard_handler` to `install_keyboard_handler` it fits the naming convention of the rest of the project better.